### DISCUSSION TITLE
Welcome to btp-service-metadata Discussions!

### DISCUSSION THREAD

**Discussion by lechnerc77** [SAP-samples/btp-service-metadata#2] [Announcements]
👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pankajjsr87** [comment]
Hi,

It's a fantastic blog with lots of metadata information about BTP Services at one place.
It would be great if we can also have Service version details along with date on which a current version has been deployed.

Thanks
Pankaj

**Reply by lechnerc77** [reply]
Hi @Pankajjsr87,

thanks for your kind feedback. 
Can elaborate a bit more on the Service version details. Do you mean GA dates and deprecation dates? The services per se are not equivalent to the APIs (an api.sap.com) used to interact with the services. The APIs follow the semantic versioning schema. For the services this makes only limited sense as they follow a continuous delivery model.

Best, Christian

**Reply by flop36** [reply]
Hi @lechnerc77 & @Pankajjsr87,

interesting discussion, thank you.
I think it would be quite interesting to see the APIs - including currently active semantic version - that can be used to interact with each service. What do you think?

Thanks and kind regards,
Flo

**Reply by lechnerc77** [reply]
Sounds like a reasonable use case. Would you mind to open an [issue](https://github.com/SAP-samples/btp-service-metadata/issues/new?assignees=&labels=enhancement&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%3Ctitle%3E) for that request? 

Best,
Christian

**Reply by flop36** [reply]
Sure, I am happy to do so!

Kind regards,
Flo

**Reply by flop36** [reply]
Hey @lechnerc77,
I created the [issue](https://github.com/SAP-samples/btp-service-metadata/issues/14)
Please let me know if you have any questions.

Thanks
Flo

**Reply by lechnerc77** [reply]
Thank you for opening the issue and for the offer: in case of questions, we come back to you!

Best,
Christian

**Reply by otavionunes** [reply]
Thanks for making that data available. I've been using Selenium to web scrape the download button at the SAP Discovery Center page. Is there any plan to make it an API instead of a repo?
